OpenSim/Application.swift

```diff
@@ -24,12 +24,12 @@ final class Application {
     static let sizeDispatchQueue = DispatchQueue(label: "com.pop-tap.size", attributes: .concurrent, target: nil)
     
     var sandboxUrl: URL? {
-        guard let url = device.containerURLForApplication(self),
-            FileManager.default.fileExists(atPath: url.path)
+        guard let safeurl = device.containerURLForApplication(self),
+            FileManager.default.fileExists(atPath: safeurl.path)
             else {
-                return nil
+                return url
```

Comment:
Thank for the reply. I'll look into the issue you mentioned. Meanwhile, if you are looking into a way for exposing the app url, I would suggest writing a separate action.